server/server.go

```diff
@@ -455,19 +455,20 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	cmdFlags := fmt.Sprintf("tag %s, repo %s, commitSHA %s, force %v", tag, repo, commitSHA, force)
+	msg := fmt.Sprintf("Tag created, flags %s. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", cmdFlags)
 	if err := createTag(ctx, client, Cfg.GithubOrg, repo, tag, commitSHA); errors.Is(err, ErrTagExists) {
 		if !force {
-			WriteErrorResponse(w, NewError(fmt.Errorf("Tag %s already exists, not generating any artifacts. Use --force to regenerate artifacts.", tag).Error(), nil))
+			WriteErrorResponse(w, NewError(fmt.Errorf("Tag already exists, details %s. Not generating any artifacts. Use --force to regenerate artifacts. ", cmdFlags).Error(), nil))
```

Comment:
Initially I was thinking to include the flags here as well, but thinking more about it, since we'll include the flags in the main message, maybe it's not required here. So lets skip.